cvat-ui/src/components/annotation-page/standard-workspace/controls-side-bar/draw-shape-popover.tsx

```diff
@@ -117,7 +113,15 @@ function DrawShapePopoverComponent(props: Props): JSX.Element {
                         </Col>
                         <Col span={10}>
                             <InputNumber
-                                onChange={onChangePoints}
+                                onChange={(value: number | undefined) => {
+                                    if (typeof (value) === 'number') {
+                                        onChangePoints(Math.floor(
+                                            clamp(value, minimumPoints, Number.MAX_SAFE_INTEGER),
```

Comment:
BTW, why is there two implementation of the clamp function? One in `utils/math.ts` other in `cvat-core/src/api.js`.

Reply:
Despite everything ``core`` and ``ui`` are independent modules and can be divided in different repositories in the future